easybuild/easyconfigs/j/jax/jax-0.2.18-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'jax'
+version = '0.2.18'
+
+homepage = 'https://pypi.python.org/pypi/jax'
+description = """Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and
+more"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/google/jax/archive/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = [
+    'jax-0.2.18-correct_libraries.patch',
+]
+
+preinstallopts = 'python3 build/build.py '
+preinstallopts += '--bazel_startup_options="--output_user_root=%(builddir)s" '
+preinstallopts += ' --bazel_path="$EBROOTBAZEL/bin/bazel" && '
+preinstallopts += 'pip install dist/*.whl && '
```

Comment:
```suggestion
preinstallopts += 'pip install --prefix %(installdir)s dist/*.whl && '
```

Reply:
BTW: @boegel I think it makes sense to enhance PythonPackage to be able to specify the location for pip. Because we add quite some useful stuff to the pip command such as `--no-deps --ignore-installed` etc. which is lost here. But we can't use PythonPackage to install this whl file yet